docs/hsi.md

```diff
@@ -372,21 +372,39 @@ To meet HSI-1 on systems that run this test, the result must be `not-enabled`. *
 - [Chipsec 4xxlp register definitions](https://github.com/chipsec/chipsec/blob/master/chipsec/cfg/8086/pch_4xxlp.xml#L270)
 - [RISC-V EDK PCH register definitions](https://github.com/riscv/riscv-edk2-platforms/blob/85a50de1b459d1d6644a402081120770aa6dd8c7/Silicon/Intel/CoffeelakeSiliconPkg/Pch/Include/Register/PchRegsDci.h)
 
+<a id="org.fwupd.hsi.PlatformFused"></a>
+
+### [Part is fused](#org.fwupd.hsi.PlatformFused)
+
+When fuses are blown in parts from some manufacturers the hardware will enforce protections against
+tampering or accessing of certain registers.
+
+**Impact:** If using an unfused part, the platform's overall security will be decreased.
+
+**Possible results:**
+
+- `locked`: device is fused
+- `not-locked`: device is not fused
+
+To meet HSI-1 on systems that run this test, the result must be `locked`. *[v1.8.0]*
+
 <a id="org.fwupd.hsi.IntelDci.Locked"></a>
+<a id="org.fwupd.hsi.PlatformDebugLocked"></a>
 
-### [Intel DCI](#org.fwupd.hsi.IntelDci.Locked)
+### [Part is debug locked](#org.fwupd.hsi.PlatformDebugLocked)
 
-Newer Intel CPUs support debugging over USB3 via a proprietary Direct Connection Interface (DCI) with the use of off-the-shelf hardware.
+Some devices support a concept of whether a part has been unlocked for debugging using proprietary hardware.  Such parts allow access to registers that are typically restricted when parts are fused.
```

Comment:
I do think we need to refer to things like DCI in these descriptions; if the ODM gets a `PlatformDebugLocked` failure they need to be able to correlate this back to something like `DCI` so they know what needs fixing. I think you can also replace *ODM* with *Security Researcher` too.

Reply:
I've modified it, can you take another look?